src/test/unit_tests/test_namespace_fs.js

```diff
@@ -205,8 +205,8 @@ mocha.describe('namespace_fs', function() {
             console.log('create_object_upload response', inspect(create_res));
 
             const obj_id = create_res.obj_id;
-            const num_parts = 10;
-            const part_size = 1024 * 1024;
+            const num_parts = 100;
+            const part_size = 10 * 1024 * 1024;
```

Comment:
I don't think this is a good idea for unit tests in memory. I would look for another way to test.